agents/insurance-solution/agent/api-schema/gather_evidence.json

```diff
@@ -0,0 +1,52 @@
+{"openapi": "3.0.0",
+    "info": {
+        "title": "Insurance Claim Evidence Gathering API",
+        "version": "1.0.0",
+        "description": "API for gathering evidence like driver's license, vehicle registration, and vehicle and accident photos for an insurance claim with pending documents"
+    },
+    "paths": {
+        "/claims/{claimId}/gather-evidence": {
```

Comment:
avoid using claim and other words that could be returned to the user query in the api paths to avoid REDACTIONS in the output